install/php.sh

```diff
@@ -0,0 +1,13 @@
+# install php with basic extensions
+sudo apt -y install php8.3 php8.3-{curl,apcu,intl,mbstring,opcache,pgsql,mysql,sqlite3,redis,xml,zip}
```

Comment:
- Why install a specific version instead of just 'php'? 
- This will also install apache. Maybe replace `php8.3` with `php-fpm php-cli` ?

Reply:
to my knowledge, apache is not included in PHP. PHP8.3 is the current LTS version.

Reply:
Without a version it would also work in the next LTS version. Part of apache is a dependency: [PHP](https://packages.ubuntu.com/noble/php) -> [PHP8.3](https://packages.ubuntu.com/noble/php8.3) -> [libapache2-mod-php8.3](https://packages.ubuntu.com/noble/libapache2-mod-php8.3) -> [apache2-bin](https://packages.ubuntu.com/noble/apache2-bin). Not sure if this is a problem for some people. It is just a suggestion I found [here](https://askubuntu.com/questions/1316542/how-to-install-multiple-php-version-without-apache) after trying it on a new Omakub installation

Reply:
that's what I personnally use, yes, but specifically because it offers several php versions, I didn't realize it had fewer dependencies too, which is nice. 

As for the LTS, I don't know. Ruby is fixed to 3.3 not latest. PHP 8.4 is going to be released in november, it's quite an easy update to do, LTS are updated every year. It's again a matter of preference, I can't speak for all PHP devs.

Reply:
Not sure if it has fewer dependencies but your current PR looks good to me. You have a good point about the fixed versions. I also can't speak for all PHP devs :-)

Reply:
Looks like you have to specify a version when using that repository:
![image](https://github.com/basecamp/omakub/assets/10061147/99e19f46-89ee-474f-8185-8d19fb1f6c30)

Maybe you had the better solution from the start. This works:
`sudo apt -y install php8.3 php8.3-{curl,apcu,intl,mbstring,opcache,pgsql,mysql,sqlite3,redis,xml,zip}`

You can remove the version when using the default repository, but I also think adding the ondrej repository is the best solution. 

The Symfony steps are removed, but they work after that and `symfony check:requirements` is successful.

I new Laravel application also works.